htdocs/expedition/list.php

```diff
@@ -149,7 +149,7 @@
 // Extra fields
 include DOL_DOCUMENT_ROOT.'/core/tpl/extrafields_list_array_fields.tpl.php';
 
-$object->fields = dol_sort_array($object->fields, 'position');
+$object->array_options = dol_sort_array($object->array_options, 'position');
```

Comment:
According to our template code into   modulebuilder/template/myobject_list.php, old code seems good.
Problem here is that the $object = Expedition does not has the ->$fields array.
In a future all objects must have this array.
So i pushed a different fix by adding just 
	public $fields = array();
into Expedition class. In a future we will have to replace it with the ->fields that describe all fields of the object.